tests/pipeline/ai/test_fall_detect.py

```diff
@@ -79,6 +79,28 @@ def sample_callback(image=None, inference_result=None, **kwargs):
     output = _OutPipeElement(sample_callback=sample_callback)
     fall_detector.connect_to_next_element(output)
 
+    img_1 = _get_image(file_name='fall_img_1.png')
+    img_2 = _get_image(file_name='fall_img_3.png')
+    fall_detector.receive_next_sample(image=img_1)
+    fall_detector.receive_next_sample(image=img_2)
+
+    assert not result
+    
+def test_fall_detection_case_2():
+    """Expected to detect a fall as key-points are detected."""
+    config = _fall_detect_config()
+    result = None
+
+    def sample_callback(image=None, inference_result=None, **kwargs):
+        nonlocal result
+        result = inference_result
+
+    fall_detector = FallDetector(**config)
+
+    output = _OutPipeElement(sample_callback=sample_callback)
+    
+    fall_detector.connect_to_next_element(output)
+
     img_1 = _get_image(file_name='fall_img_1.png')
     img_2 = _get_image(file_name='fall_img_2.png')
```

Comment:
@bhavikapanara I see that fall_img_2.png is a person leaning over. Is the angle between standing and leaning over 60'? Could you please attache images with pose lines overlay in each image?

Let's add a negative test case that covers images where the poses are detected but the angle is less than the required fall factor.

Reply:
@bhavikapanara thoughts on this one?

Reply:
Yes...just commit negative test case

Reply:
![fall_img_1](https://user-images.githubusercontent.com/19815572/101804629-d05e1980-3b37-11eb-9b61-32ad3e89537c.jpg)
![fall_img_2](https://user-images.githubusercontent.com/19815572/101804646-d5bb6400-3b37-11eb-93fd-fc292ecba0fb.jpg)

Here are the key-points of the shoulder and hip : [[left shoulder, left hip], [right shoulder, right hip]]

First Image      :  `[[[436, 148], [431, 211]], [[401, 151], [416, 217]]]`
Second Image :  `[[[309, 209], [336, 278]], [[274, 211], [306, 280]]]`

Reply:
OK, this looks right. Seems like the angle is less than 60', no? What number do you get with your formula?

Reply:
I also have doubts. Can you please look at this [Colab file](https://colab.research.google.com/drive/1UdrVwxJb3v6dqmPNmD5_UxrJQPo0VzGF?usp=sharing)?

Reply:
OK, let me look closer. Something is off with the calculations.

Reply:
OK, please look at the suggested corrections in the colab file and see you get better results on your end. The main issue was with using atan which does not give us the correct quadrant of an angle, whereas atan2 does.

More here: https://stackoverflow.com/questions/2676719/calculating-the-angle-between-the-line-defined-by-two-points

Reply:
The main change:

```
import math

def calculate_angle(p):
    x1, y1 = p[0][0][0], p[0][0][1]
    x2, y2 = p[0][1][0], p[0][1][1]
    x3, y3 = p[1][0][0], p[1][0][1]
    x4, y4 = p[1][1][0], p[1][1][1]
    
    print("x1, y1 : ",x1, y1)
    print("x2, y2 : ", x2, y2)
    print("x3, y3 : ", x3, y3)
    print("x4, y4 : ", x4, y4)
    
    # since y coordinates in screen and pose estimation are reversed
    # (higher values are lower on the screen), we need to reverse the y delta too

    theta1 = math.degrees(math.atan2(y2-y1, x1-x2))
    theta2 = math.degrees(math.atan2(y4-y3, x3-x4))
        
    print("theta1 : ", theta1)
    print("theta2 : ", theta2)

    theta = abs(theta1-theta2)
    return theta
```